src/views/htmlcontent/src/js/components/app.component.ts

```diff
@@ -37,12 +37,18 @@ export interface IGridDataSet {
 // tslint:disable:max-line-length
 const template = `
 <div class="fullsize vertBox">
-    <div *ngIf="dataSets.length > 0" id="resultspane" class="boxRow header collapsible" [class.collapsed]="!resultActive" (click)="toggleResultsPane()">
+    <div *ngIf="dataSets.length > 0" role="button" id="resultspane" tabIndex="0" class="boxRow header collapsible"
+        [class.collapsed]="!resultActive"
+        (click)="toggleResultsPane()"
+        [attr.aria-label]="Constants.resultPaneLabel"
+        [attr.aria-expanded]="!resultActive">
         <span> {{Constants.resultPaneLabel}} </span>
         <span class="shortCut"> {{resultShortcut}} </span>
     </div>
-    <div id="results" *ngIf="renderedDataSets.length > 0" class="results vertBox scrollable"
-         (onScroll)="onScroll($event)" [scrollEnabled]="scrollEnabled" [class.hidden]="!resultActive">
+    <div id="results" tabIndex="0" *ngIf="renderedDataSets.length > 0" class="results vertBox scrollable"
```

Comment:
you are enabling keyboard focus on this element, but i don't see any action you can do here, am i missing something?

Reply:
it wasn't able to get focus earlier because `div`s don't support focus unless they have a tabIndex. You couldn't reach just the header just with keyboard.